src/endpoints/transactions/transaction.service.ts

```diff
@@ -152,7 +152,11 @@ export class TransactionService {
     const elasticTransactions = await this.indexerService.getTransactions(filter, pagination, address);
 
     let transactions: TransactionDetailed[] = [];
-    transactions = elasticTransactions.map(x => ApiUtils.mergeObjects(new TransactionDetailed(), x));
+         transactions = elasticTransactions.map(x => {
+           const transaction = ApiUtils.mergeObjects(new TransactionDetailed(), x);
+           transaction.relayedVersion = x.version;
```

Comment:
` relayedVersion` and `transaction version` are different concepts, just to double check if everything is all right.

Reply:
reverted

Reply:
done